library/src/main/camera2/com/google/android/cameraview/Camera2.java

```diff
@@ -385,7 +407,6 @@ void setFacing(int facing) {
         mFacing = facing;
         if (isCameraOpened()) {
             stop();
-            start();
```

Comment:
Hmm, this is odd. I'll have a look in the commits to see when that was added initially.